sdks/typescript/package.json

```diff
@@ -36,7 +36,7 @@
   },
   "dependencies": {
     "@google-cloud/pubsub": "^2.19.4",
-    "@grpc/grpc-js": "^1.8.8",
+    "@grpc/grpc-js": "^1.4.6",
```

Comment:
I'm kinda confused by this one: isn't `^1.4.6` just a more permissive superset of `^1.8.8`? Should we be pinning to an explicit version instead of using the `^`?

In semver `^1.4.6` is equivalent to `>=1.4.6, <2.0.0`

Reply:
Actually, I'm not sure why this fixed things then. I'm thinking maybe I already had an older grpc in the starter project which is why not forcing the lower bound didn't break things. (I have to admit I'm a bit out of depth with these dependency issues here. It's also curious that the `TypeScript xlang Tests` didn't fail on the depend bot PR. The error I'm struggling with manifests as https://github.com/grpc/grpc-node/issues/2411 ; open to any ideas here.)

Reply:
Probably its actually the package-lock changes that fixed things. Specifically, `"node_modules/@grpc/grpc-js"` went from `1.8.8` to `1.4.6`. But that kinda just means you got lucky 😄 we should change this from `^1.4.6` to `1.4.6` if we're sensitive to the version changing (or at least `~1.4.6` if we don't care about the patch version changing). See https://devhints.io/semver

If we don't do that, the next time someone runs `npm install` (if for example they change a different package), this is liable to change.

Reply:
Yeah. Updated to restrict to the patch version.